app/sidebar.py

```diff
@@ -119,13 +119,32 @@ def make_sidebar(api: PtxboaAPI):
         horizontal=True,
         help=read_markdown_file("md/helptext_sidebar_water_source.md"),
     )
-    st.session_state["transport"] = st.sidebar.radio(
-        "Mode of transportation (for selected supply country):",
-        ["Ship", "Pipeline"],
-        horizontal=True,
-        help=read_markdown_file("md/helptext_sidebar_transport.md"),
-        index=1,  # 'Pipeline' as default
-    )
+
+    # determine transportation distance, only allow pipeline if <6000km:
+    res = api.get_input_data(st.session_state["scenario"])
+    distance = res.loc[
+        (res["source_region_code"] == st.session_state["region"])
+        & (res["target_country_code"] == st.session_state["country"])
+        & (res["parameter_code"] == "shipping distance"),
+        "value",
+    ][0]

```

Comment:
this line triggers a warning: 
```
Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`
```

```suggestion
    ].iloc[0]


```